src/main/java/com/gluonhq/substrate/target/DarwinTargetConfiguration.java

```diff
@@ -61,9 +63,16 @@ String getAdditionalSourceFileLocation() {
         return answer;
     }
 
+    @Override
```

Comment:
semantically, it sounds strange that the "common" libraries can be overridden by platform-specific implementations.
Unless you refer to "common" as for all applications?

Reply:
Initially these libraries were part of the AbstractTarget, so every target were using them, that's why "common", but since it can be overridden and modified per target, it might required a different name.

Reply:
yes, changing the name and adding some basic documentation is useful.
Also, some libraries are really hard fixed to be on all platforms (at least libjava.a is hard needed on all JNI-based platforms) so it might make sense to have those common. Not a showstopper though

Reply:
This can be tackled as well as part of #102